app/src/main/java/net/squanchy/navigation/view/BottomNavigationPagerAdapter.java

```
@@ -0,0 +1,40 @@
+package net.squanchy.navigation.view;
+
+import android.content.Context;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.novoda.viewpageradapter.ViewPagerAdapter;
+
+import net.squanchy.schedule.ScheduleView;
+
+public class BottomNavigationPagerAdapter extends ViewPagerAdapter<View> {
+
+    private static final int SCHEDULE_POSITION = 0;
+    private static final int FAVOURITES_POSITION = 1;
+    private static final int TWEETS_POSITION = 2;
+    private static final int VENUE_POSITION = 3;
+    private final Context context;
+
+    public BottomNavigationPagerAdapter(Context context) {
+        this.context = context;
+    }
+
+    @Override
+    protected View createView(ViewGroup container, int position) {
+        if (position == SCHEDULE_POSITION) {
+            return ScheduleView.inflate(context, container);
+        }
+        return null;
```

I'd rather throw in this case, better than risking to fail silently